opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/SSLCertMonitor.adoc

```diff
@@ -20,6 +19,10 @@ The output shows you the time range a certificate is valid:
 
 You can configure a threshold in days applied on the `notAfter` date.
 
+While the monitor is mainly useful for plain SSL sockets, the monitor does provide limited support for STARTTLS protocols by providing the user with the ability to specify a STARTTLS message to be sent prior to the SSL negotiation and a regular expression to match to the response received from the server. An additional preliminary message and response regular expression pair is available for protocols that require it (such as XMPP).
```

Comment:
Found this one here as well - Sorry :) Can we apply the one-sentence-per line rule in this file as described in the asciidoc-recommended-practices.